packages/block-editor/src/components/colors-gradients/panel-color-gradient-settings.native.js

```diff
@@ -1,17 +1,51 @@
 /**
  * WordPress dependencies
  */
-import { PanelBody, UnsupportedFooterControl } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
+import {
+	ColorControl,
+	BottomSheetConsumer,
+	PanelBody,
+} from '@wordpress/components';
 
-const PanelColorGradientSettings = () => {
+export default function PanelColorGradientSettings( { settings, title } ) {
 	return (
-		<PanelBody>
-			<UnsupportedFooterControl
-				label={ __( 'Color settings are coming soon.' ) }
-				separatorType="none"
-			/>
+		<PanelBody title={ title }>
+			<BottomSheetConsumer>
+				{ ( { onReplaceSubsheet } ) =>
+					settings.map(
+						(
+							{
+								onColorChange,
+								colorValue,
+								onGradientChange,
+								gradientValue,
+								label,
+							},
+							index
+						) => (
+							<ColorControl
+								onPress={ () => {
+									onReplaceSubsheet( 'Color', {
```

Comment:
Shouldn't this be:

```suggestion
									onReplaceSubsheet( colorsUtils.subsheets.color, {
```

Reply:
I'm not sure about that since `colorsUtils` live in a separate package (@wordpress/components), and I think we shouldn't export them.

Reply:
Makes sense 👍